main-window.py

```diff
@@ -214,16 +214,13 @@ def __init__(self, window, column_names, graph_page, dm):
         self.add_widgets()
 
         if self.graph_page.time_series is None:
-            self.time_series = TimeSeries(self.graph_page.interior, self.dm)
+            self.time_series = TimeSeries(self.graph_page, self.dm)
```

Comment:
If I'm not mistaken, this had to use the .interior instead of the graph_page itself due to certain things using pack and others using grid for layout. Are you seeing something different with this?

Reply:
With the changes I made, I needed access to the graph page object. Any place where the interior was needed I added it in the child components